recipes/evermore/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "evermore" %}
+{% set version = "0.2.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/evermore-{{ version }}.tar.gz
+  sha256: a56b6321f0ed22c1eaeb2679baf2d7b12d0c9c8374486e46b11b97cfd4f6c2c8
+
+build:
+  noarch: python
```

Comment:
Windows builds failing are expected as JAX is not supported on this platform. If my understanding is correct it's ok to leave failing windows builds on "noarch".

Reply:
I told @pfackeldey this given https://github.com/conda-forge/staged-recipes/pull/26293#discussion_r1594838679.